src/helpers/ModuleEntityLib.sol

```diff
@@ -2,11 +2,13 @@
 pragma solidity ^0.8.20;
 
 import {ModuleEntity} from "../interfaces/IModularAccount.sol";
+// ModuleEntity is a packed representation of a module function
+// Layout:
+// 0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA________________________ // Address
+// 0x________________________________________BBBBBBBB________________ // Entity ID
+// 0x________________________________________________0000000000000000 // unused
```

Comment:
Also maybe newline before this, and consider making this a NatSpec @dev comment?

Reply:
Oh I see that this is for `ModuleEntity` imported above. Is this necessary? IModularAccount already describes it.

Reply:
It's repeated in both `IModularAccount` and each of the type encoding / decoding libraries (`ValidationConfigLib`, `HookConfigLib`) so I added it here too.